clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -233,7 +235,7 @@ private class OffsetFetchRequestState extends RequestState {
         public OffsetFetchRequestState(final Set<TopicPartition> partitions,
                                        final GroupState.Generation generation,
                                        final long retryBackoffMs) {
-            super(retryBackoffMs);
+            super(logContext, retryBackoffMs);
```

Comment:
I was thinking, to be consistent, maybe we should just pass a config in like other request manager, instead of parsing for a specific config.

Reply:
This code has been sufficiently changed that I'm not sure I understand the comment 😄